app/src/main/java/net/grandcentrix/blutufintegration/ui/list/ListFragment.kt

```diff
@@ -57,15 +57,15 @@ class ListFragment : Fragment(), DevicesAdapter.OnClickActions {
             binding.swipToRefresh.isRefreshing = false
         }
 
-
         viewModel.uiModel.observe(viewLifecycleOwner, { uiModel -> updateUi(uiModel) })
         viewModel.selectedDevice.observe(viewLifecycleOwner, { device -> updateDevice(device) })
+        viewModel.scanState.observe(
+            viewLifecycleOwner,
+            { isScanning -> updateScanState(isScanning) })
```

Comment:
In Kotlin it is possible and common to move a lambda out of the parameter list and append it after the closing bracket.
```suggestion
        viewModel.scanState.observe(viewLifecycleOwner) { isScanning ->
            updateScanState(isScanning) 
        }
```